go.mod

```diff
@@ -50,11 +50,9 @@ require (
 	github.com/docker/distribution v2.8.1+incompatible // indirect
 	github.com/felixge/httpsnoop v1.0.2 // indirect
 	github.com/google/btree v1.1.2 // indirect
-	github.com/hashicorp/go-uuid v1.0.2 // indirect
-	github.com/jhump/protoreflect v1.13.1-0.20220928232736-101791cb1b4c // indirect
 	github.com/kkHAIKE/contextcheck v1.1.3 // indirect
 	github.com/maratori/testableexamples v1.0.0 // indirect
-	github.com/onsi/ginkgo v1.16.4 // indirect
+	github.com/nxadm/tail v1.4.8 // indirect
```

Comment:
Should we be updating `osmoutils`?

Reply:
This cone be done in this PR in progress: https://github.com/osmosis-labs/osmosis/pull/4038